.travis.yml

```diff
@@ -10,3 +10,4 @@ install:
   - pip install -r requirements.txt
 script:
   - flake8 --max-line-length=99 testfm tests
+  - make test-docstrings
```

Comment:
Add uuid-check as well

Reply:
@ntkathole before calling test-docstrings it will call uuid-check.  `test-docstrings: uuid-check`